packages/mikro-orm/src/models.ts

```diff
@@ -0,0 +1,162 @@
+import { randomUUID } from "crypto"
```

Comment:
This would only work in Node 14 and upwards. I think for now using `uuid` as an extra dependency is a way to go.